src/m365/spo/commands/file/file-checkin.ts

```diff
@@ -109,6 +110,10 @@ class SpoFileCheckinCommand extends SpoCommand {
     this.optionSets.push({ options: ['url', 'id'] });
   }
 
+  protected getExcludedOptionsWithUrls(): string[] | undefined {
+    return ['url'];
```

Comment:
Have you double checked that this isn't changing the existing behavior in a backwards incompatible way and we won't need to postpone merging this PR until v7?

Reply:
As far as I know, we are not breaking anything. Tested every single command with both server relative URLs and site relative URLs.

Reply:
Correct. My point was, that before this change we were expanding server-relative URLs to absolute. If however this change still allows users to pass server-relative URLs, then all is good. I thought I'd double check before proceeding

Reply:
No problem, better check too much than too little